requirements.txt

```
@@ -19,6 +19,7 @@ email-validator==1.1.3
 factory-boy==3.2.1
 Faker==12.0.0
 Flask==2.0.2
+Flask-APScheduler==1.12.1
```

```suggestion
Flask-APScheduler==1.12.3
```
Going back to 1.12.3 again. `FLASK_ENV` must be set to `production`; `development` will not work.